<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetDatabaseNameWindow</class>
 <widget class="QDialog" name="GetDatabaseNameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Wpisz nazwę nowej bazy </string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pamiętaj, że nazwa musi być inna, od nazw już istniejących baz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="come_back_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Powrót</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
